tests/PINCache.xcodeproj/xcshareddata/xcschemes/iOS.xcscheme

```diff
@@ -28,7 +28,26 @@
       shouldUseLaunchSchemeArgsEnv = "YES"
       buildConfiguration = "Debug">
       <Testables>
+         <TestableReference
```

Comment:
I've added ability to run tests from iOS scheme by clicking cmd+u. 
Needs to be removed.